src/shared/managers/ServiceWorkerManager.ts

```diff
@@ -174,6 +174,8 @@ export class ServiceWorkerManager {
     });
   }
 
+  // Returns true if a the OneSignal service worker can't be installed
```

Comment:
"Returns true if a" - grammar

Also, this logic doesn't match the function. `shouldInstallWorker` should return `true` if it can and should be installed.

Reply:
Good catch on both, fixed